plugins/grpc/csharp/v1.65.0/build.csproj

```diff
@@ -4,6 +4,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Grpc.Net.Common" Version="2.63.0" />
+    <PackageReference Include="Grpc.Net.Client" Version="2.63.0" />
```

Comment:
technically these don't need to be here as they aren't used in compilation, but we're currently testing to ensure that they match. Ideally we'd separate out this dep from the others, but .. 🤷‍♂️? Shouldn't matter much.